### ISSUE TITLE
Cannot build android in 6.13.0

### ISSUE THREAD

**Issue by Thaina** [firebase/quickstart-unity#619]
### Please fill in the following fields:
Unity editor version: 
Firebase Unity SDK version: 6.13.0
Firebase plugins in use : Auth FireStore Analytics Crashlytics Messaging DynamicLinks
Additional SDKs you are using: Facebook AdMob GooglePlayGames
Platform you are using the Unity editor on : Windows 
Platform you are targeting : Android
Scripting Runtime : IL2CPP

### Please answer the following, if applicable:
Have you been able to reproduce this issue with just the Firebase Unity quickstarts (this GitHub project)?
Don't know

What's the issue repro rate?
100%

### Please describe the issue here:
Install 6.13 with EDM4U UPM and build can be run in editor properly but cannot build android with below error

```
Uploading Crash Report
CommandInvokationFailure: Gradle build failed. 
D:\2018.4.19f1\Editor\Data\PlaybackEngines\AndroidPlayer/Tools\OpenJDK\Windows\bin\java.exe -classpath "D:\2018.4.19f1\Editor\Data\PlaybackEngines\AndroidPlayer\Tools\gradle\lib\gradle-launcher-5.1.1.jar" org.gradle.launcher.GradleMain "-Dorg.gradle.jvmargs=-Xmx4096m" "assembleRelease"

stderr[

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':lintVitalRelease'.
> Could not resolve all task dependencies for configuration ':releaseRuntimeClasspath'.
   > Could not find com.google.firebase:firebase-analytics-unity:6.13.0.
     Searched in the following locations:
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/m2repository/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.pom
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/m2repository/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.jar
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/google/m2repository/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.pom
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/google/m2repository/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.jar
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/android/m2repository/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.pom
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/android/m2repository/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.jar
       - https://dl.google.com/dl/android/maven2/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.pom
       - https://dl.google.com/dl/android/maven2/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.jar
       - https://jcenter.bintray.com/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.pom
       - https://jcenter.bintray.com/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Temp/gradleOut/libs/firebase-analytics-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Temp/gradleOut/libs/firebase-analytics-unity.jar
       - https://maven.google.com/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.pom
       - https://maven.google.com/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.jar
       - https://adcolony.bintray.com/AdColony/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.pom
       - https://adcolony.bintray.com/AdColony/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/GooglePlayGames/Editor/m2repository/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.pom
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/GooglePlayGames/Editor/m2repository/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/Firebase/m2repository/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.pom
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/Firebase/m2repository/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.jar
       - file:/C:/Users/Admin/.m2/repository/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.pom
       - file:/C:/Users/Admin/.m2/repository/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.jar
       - https://repo.maven.apache.org/maven2/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.pom
       - https://repo.maven.apache.org/maven2/com/google/firebase/firebase-analytics-unity/6.13.0/firebase-analytics-unity-6.13.0.jar
     Required by:
         project :
   > Could not find com.google.firebase:firebase-auth-unity:6.13.0.
     Searched in the following locations:
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/m2repository/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.pom
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/m2repository/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.jar
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/google/m2repository/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.pom
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/google/m2repository/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.jar
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/android/m2repository/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.pom
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/android/m2repository/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.jar
       - https://dl.google.com/dl/android/maven2/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.pom
       - https://dl.google.com/dl/android/maven2/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.jar
       - https://jcenter.bintray.com/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.pom
       - https://jcenter.bintray.com/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Temp/gradleOut/libs/firebase-auth-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Temp/gradleOut/libs/firebase-auth-unity.jar
       - https://maven.google.com/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.pom
       - https://maven.google.com/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.jar
       - https://adcolony.bintray.com/AdColony/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.pom
       - https://adcolony.bintray.com/AdColony/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/GooglePlayGames/Editor/m2repository/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.pom
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/GooglePlayGames/Editor/m2repository/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/Firebase/m2repository/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.pom
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/Firebase/m2repository/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.jar
       - file:/C:/Users/Admin/.m2/repository/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.pom
       - file:/C:/Users/Admin/.m2/repository/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.jar
       - https://repo.maven.apache.org/maven2/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.pom
       - https://repo.maven.apache.org/maven2/com/google/firebase/firebase-auth-unity/6.13.0/firebase-auth-unity-6.13.0.jar
     Required by:
         project :
   > Could not find com.google.firebase:firebase-crashlytics-unity:6.13.0.
     Searched in the following locations:
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/m2repository/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.pom
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/m2repository/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.jar
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/google/m2repository/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.pom
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/google/m2repository/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.jar
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/android/m2repository/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.pom
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/android/m2repository/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.jar
       - https://dl.google.com/dl/android/maven2/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.pom
       - https://dl.google.com/dl/android/maven2/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.jar
       - https://jcenter.bintray.com/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.pom
       - https://jcenter.bintray.com/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Temp/gradleOut/libs/firebase-crashlytics-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Temp/gradleOut/libs/firebase-crashlytics-unity.jar
       - https://maven.google.com/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.pom
       - https://maven.google.com/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.jar
       - https://adcolony.bintray.com/AdColony/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.pom
       - https://adcolony.bintray.com/AdColony/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/GooglePlayGames/Editor/m2repository/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.pom
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/GooglePlayGames/Editor/m2repository/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/Firebase/m2repository/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.pom
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/Firebase/m2repository/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.jar
       - file:/C:/Users/Admin/.m2/repository/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.pom
       - file:/C:/Users/Admin/.m2/repository/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.jar
       - https://repo.maven.apache.org/maven2/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.pom
       - https://repo.maven.apache.org/maven2/com/google/firebase/firebase-crashlytics-unity/6.13.0/firebase-crashlytics-unity-6.13.0.jar
     Required by:
         project :
   > Could not find com.google.firebase:firebase-dynamic-links-unity:6.13.0.
     Searched in the following locations:
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/m2repository/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.pom
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/m2repository/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.jar
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/google/m2repository/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.pom
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/google/m2repository/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.jar
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/android/m2repository/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.pom
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/android/m2repository/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.jar
       - https://dl.google.com/dl/android/maven2/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.pom
       - https://dl.google.com/dl/android/maven2/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.jar
       - https://jcenter.bintray.com/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.pom
       - https://jcenter.bintray.com/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Temp/gradleOut/libs/firebase-dynamic-links-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Temp/gradleOut/libs/firebase-dynamic-links-unity.jar
       - https://maven.google.com/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.pom
       - https://maven.google.com/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.jar
       - https://adcolony.bintray.com/AdColony/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.pom
       - https://adcolony.bintray.com/AdColony/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/GooglePlayGames/Editor/m2repository/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.pom
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/GooglePlayGames/Editor/m2repository/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/Firebase/m2repository/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.pom
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/Firebase/m2repository/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.jar
       - file:/C:/Users/Admin/.m2/repository/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.pom
       - file:/C:/Users/Admin/.m2/repository/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.jar
       - https://repo.maven.apache.org/maven2/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.pom
       - https://repo.maven.apache.org/maven2/com/google/firebase/firebase-dynamic-links-unity/6.13.0/firebase-dynamic-links-unity-6.13.0.jar
     Required by:
         project :
   > Could not find com.google.firebase:firebase-firestore-unity:6.13.0.
     Searched in the following locations:
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/m2repository/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.pom
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/m2repository/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.jar
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/google/m2repository/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.pom
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/google/m2repository/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.jar
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/android/m2repository/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.pom
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/android/m2repository/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.jar
       - https://dl.google.com/dl/android/maven2/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.pom
       - https://dl.google.com/dl/android/maven2/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.jar
       - https://jcenter.bintray.com/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.pom
       - https://jcenter.bintray.com/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Temp/gradleOut/libs/firebase-firestore-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Temp/gradleOut/libs/firebase-firestore-unity.jar
       - https://maven.google.com/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.pom
       - https://maven.google.com/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.jar
       - https://adcolony.bintray.com/AdColony/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.pom
       - https://adcolony.bintray.com/AdColony/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/GooglePlayGames/Editor/m2repository/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.pom
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/GooglePlayGames/Editor/m2repository/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/Firebase/m2repository/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.pom
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/Firebase/m2repository/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.jar
       - file:/C:/Users/Admin/.m2/repository/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.pom
       - file:/C:/Users/Admin/.m2/repository/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.jar
       - https://repo.maven.apache.org/maven2/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.pom
       - https://repo.maven.apache.org/maven2/com/google/firebase/firebase-firestore-unity/6.13.0/firebase-firestore-unity-6.13.0.jar
     Required by:
         project :
   > Could not find com.google.firebase:firebase-messaging-unity:6.13.0.
     Searched in the following locations:
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/m2repository/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.pom
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/m2repository/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.jar
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/google/m2repository/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.pom
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/google/m2repository/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.jar
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/android/m2repository/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.pom
       - file:/D:/2020.1.0b2/Editor/Data/PlaybackEngines/AndroidPlayer/SDK/extras/android/m2repository/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.jar
       - https://dl.google.com/dl/android/maven2/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.pom
       - https://dl.google.com/dl/android/maven2/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.jar
       - https://jcenter.bintray.com/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.pom
       - https://jcenter.bintray.com/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Temp/gradleOut/libs/firebase-messaging-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Temp/gradleOut/libs/firebase-messaging-unity.jar
       - https://maven.google.com/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.pom
       - https://maven.google.com/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.jar
       - https://adcolony.bintray.com/AdColony/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.pom
       - https://adcolony.bintray.com/AdColony/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/GooglePlayGames/Editor/m2repository/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.pom
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/GooglePlayGames/Editor/m2repository/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.jar
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/Firebase/m2repository/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.pom
       - file:/D:/Users/Admin/Documents/mycompany/myapp/ss/app/mygame/Assets/Firebase/m2repository/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.jar
       - file:/C:/Users/Admin/.m2/repository/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.pom
       - file:/C:/Users/Admin/.m2/repository/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.jar
       - https://repo.maven.apache.org/maven2/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.pom
       - https://repo.maven.apache.org/maven2/com/google/firebase/firebase-messaging-unity/6.13.0/firebase-messaging-unity-6.13.0.jar
     Required by:
         project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 45s
]
```

Also cause error in ios (can build but error in runtime), will log it soon

**Comment by alexames** [comment]
It sounds like you may have missed a step when setting up your project with firebase. I suggest starting with a clean copy of this repository, and follow the steps in the readme file exactly to ensure you get things set up correctly. If you're still seeing issues, we can try to figure out what went wrong.

**Comment by chkuang-g** [comment]
@Thaina 

Is your project using gradle template mode?  

If so, this might be an issue with External Dependency Manager(EDM, former Play Services Resolver) which would attempt to generate `.aar` at where `.srcaar` at, which is under Packages folder (Library/PackageCache/ actually).

If this is the main cause, I would recommend you to install latest Firebase from unitypackage for now until we fix this issue in EDM.

Shawn

**Comment by Thaina** [comment]
@chkuang-g Yes, my project using gradle, and that's what I suspect too, because firebase folder in UPM contain `m2repository`, but the gradle template still pointed to `Assets/Firebase/m2repository` which has very difference folder structure

**Comment by Thaina** [comment]
@chkuang-g Not only it can't explode `.aar` in `PackageCache`, the repository folder also pointed to the wrong folder. Until I have made it pointed to the right folder it then cause another error about cannot explode `.aar`

**Comment by snorrsi** [comment]
Are you working on fixing this issue?

Doesn't seem right to put this under feature request as this breaks all gradle builds. At least you should mention this in changelog.

**Comment by chkuang-g** [comment]
@snorrsi 

Yes, we are working on this on the EDM side. As a workaround, you can install Firebase using `.unitypackage` instead for now.
https://firebase.google.com/download/unity

**Comment by chkuang-g** [comment]
Hi folks,

This should be fixed in EDM 1.2.151. You should be able to find it through Unity Package Manager.

You should be able to use Firebase 6.13.0 with this fix. What you need to do is:
* Follow the steps here to install Firebase through UPM
  https://firebase.google.com/docs/unity/setup#add-sdks-upm
* Search for "External Dependency Manager" through Unity Package Manager window.
* Upgrade External Dependency Manager to any version above 1.2.151

What it does is:
* Android Resolver will copy aar and pom into repo under Assets/GeneratedLocalRepo
* You can change the default repo location in Settings.

Let us know if this works for you.

Shawn
